src/omv/boards/ARDUINO_GIGA/imlib_config.h

```diff
@@ -96,6 +96,12 @@
 
 // Enable find_apriltags() (64 KB)
 #define IMLIB_ENABLE_APRILTAGS
+//#define IMLIB_ENABLE_APRILTAGS_TAG16H5
+//#define IMLIB_ENABLE_APRILTAGS_TAG25H7
+//#define IMLIB_ENABLE_APRILTAGS_TAG25H9
+//#define IMLIB_ENABLE_APRILTAGS_TAG36H10
```

Comment:
```suggestion
```
Let's just define the enabled ones, until we have a common `imlib_config.h`

Reply:
Okay